kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/EditorController.java

```diff
@@ -580,6 +580,16 @@ public ObservableValue<Theme> themeProperty() {
         return themeProperty;
     }
 
+    /**
+     * Refresh the theme and related stylesheets in different
+     * places (content, preview, ...)
+     */
+    public void refreshTheme() {
+        EditorPlatform.Theme currentTheme = getTheme();
+        setTheme(null);
+        setTheme(currentTheme);
```

Comment:
why is setting the theme to `null` required?

Reply:
EditorController has `themeProperty` with a default theme. When the theme changes, it refresh the scene graph from the `invalidated()` method, and there are a bunch of listeners in different places (5 so far) attached to `themeProperty()` that react accordingly.

For instance, in `ContentPaneController` we have:
```
editorController.themeProperty().addListener((ov, t, t1) -> themeDidChange());
```
that ultimately calls  `EditorPlatform::getStylesheetsForTheme`, that brings in the Gluon plugin stylesheets and the updated swatch color stylesheet.

The easy way to trigger all those listeners is to force an invalidation of `editorController.themeProperty()`, and that's easily done with setting null and restoring the current theme.